frontend/src/app/main/ui/viewer/handoff/attributes/common.cljs

```diff
@@ -16,38 +16,13 @@
    [app.util.color :as uc]
    [app.common.math :as mth]
    [app.main.ui.icons :as i]
+   [app.util.code-gen :as code]
```

Comment:
For consistency... always "code" or always "cg" ?

Reply:
OK